src_c/line.c

```diff
@@ -766,21 +619,18 @@ pg_line_iterator(pgLineObject *self)
     return iter;
 }
 
-#define __LINE_GETSET_NAME(name)                                          \
```

Comment:
You need to tackle this formatting problem. You see how clean this was with Clang format? you either disable your auto format on save if on VScode or something else, but we can't have a 500 lines change of just formatting, it's making it harder to spot the changes.